### ISSUE TITLE
No global events (eg. 'before-js-reload') in node

### ISSUE THREAD

**Issue by mhuebert** [bhauman/lein-figwheel#486]
Currently in node development it is not possible to listen to global events, such as "before-js-reload". Per https://github.com/bhauman/lein-figwheel/blob/e47da1658a716f83888e5a5164ee88e59b2d8c1e/support/src/figwheel/client/utils.cljs#L29, this would require lifting the events system off the DOM.

**Comment by bhauman** [comment]
yep this is not a current feature, I'm open to a pull request here.
